packages/amplify-category-function/provider-utils/awscloudformation/cloudformation-templates/lambda-cloudformation-template.json.ejs

```diff
@@ -214,6 +222,35 @@
         }
         <% } %>
         <% } %>
+        <% if (props.cloudwatchEvent && props.cloudwatchEvent != "NONE") { %> 
+        ,"CloudWatchEvent": {
+            "Type" : "AWS::Events::Rule",
+            "Properties":{
+                "Description" : "Schedule rule for Lambda",
+                "ScheduleExpression" : "<%= props.cloudwatchRule %>",
+                "State": "ENABLED",
+                "Targets": [{
+                    "Arn": { "Fn::GetAtt": ["LambdaFunction", "Arn"] },
+                    "Id":{ 
+                        "Ref" : "LambdaFunction"
+                    }
+                }]
+            } 
+        }
+        <% } %>          
+        <% if (props.cloudwatchEvent && props.cloudwatchEvent != "NONE") { %>
```

Comment:
This if block is not required as the conditions are the same as above.

Reply:
fixed